service/service.py

```diff
@@ -1,31 +1,78 @@
 """
-My Service
+Products Service
 
-Describe what your service does here
+Paths:
+------
+GET /products - Returns a list all of the products
+GET /products/{id} - Returns the product with a given id number
+POST /products - creates a new product record in the database
+PUT /products/{id} - updates a product record in the database
+DELETE /products/{id} - deletes a product record in the database
 """
 
-import os
-import sys
-import logging
```

Comment:
@Kjabraham why are these not needed anymore? I'm going to ask you to explain a bit so I can understand things before I merge the pull request ha.

Reply:
they just weren't used anywhere in that code so we do not need to import them.

Reply:
@Kjabraham is that expected that they're not used anywhere? Is that just for now or will they possibly be used as we go further along, and so would just commenting them out make more sense than deleting them entirely? Could you give me a one-two sentence explanation of what those lines would do?

Reply:
so those three lines are importing the libraries os, sys and logging. We use them in other files (models.py and __init__.py) for example so we import them there. I do not think we will ever need them in the service.py. The os and sys libraries are used to access things on the computer. We use 'os' to access the docker container which hosts our database for example.

Reply:
Alright I will stem the continuing flow of questions and trust you that they're not needed here, and if so we can still add them back in afterwards.